### PR TITLE
refactor: Add order-preserving columns to execution outcomes

### LINKED ISSUE #31: Add order-preserving columns to execution outcomes
Blocked by https://github.com/near/nearcore/pull/3529

* `index_in_chunk`
* `executed_in_chunk_hash`
* `executed_in_block_timestamp`

### PR DESCRIPTION
**Pull request by khorolets**
Closes #31

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,5 +26,8 @@ tokio-diesel = "0.3.0"
 tracing = "0.1.13"
 tracing-subscriber = "0.2.4"
 
-near-indexer = { git = "https://github.com/nearprotocol/nearcore", rev="774ac1fe95ef6d1e1741517b7bb46f989c767c32" }
-near-crypto = { git = "https://github.com/nearprotocol/nearcore", rev="774ac1fe95ef6d1e1741517b7bb46f989c767c32" }
+#near-indexer = { git = "https://github.com/nearprotocol/nearcore", rev="774ac1fe95ef6d1e1741517b7bb46f989c767c32" }
+#near-crypto = { git = "https://github.com/nearprotocol/nearcore", rev="774ac1fe95ef6d1e1741517b7bb46f989c767c32" }
+
+near-indexer = { path = '../nearcore/chain/indexer' }
+near-crypto = { path = '../nearcore/core/crypto' }
```
Comment (frol) [inline Cargo.toml:33]: Update it

File: src/main.rs [hunk 2]
```diff
@@ -93,6 +104,9 @@ async fn listen_blocks(
     while let Some(streamer_message) = stream.recv().await {
         // Block
         info!(target: "indexer_for_explorer", "Block height {}", &streamer_message.block.header.height);
+        if streamer_message.block.header.height > 22303582 {
+            std::process::exit(0);
+        }
```
Comment (frol) [inline src/main.rs:109]: We don't want it here 😃

**Review by frol — APPROVED** [review]
LGTM! Address the comments and feel free to merge it

File: src/main.rs (hunk 2 above)
Comment (khorolets) [reply src/main.rs:109]: ouch)

**Review by frol — APPROVED** [review]

**Pull request merged**